source/scripts/Background/controllers/transactions/evm.ts

```diff
@@ -0,0 +1,147 @@
+import axios from 'axios';
+import { Chain, chains } from 'eth-chains';
+import { ethers } from 'ethers';
+import { isNil, isString } from 'lodash';
+
+import store from 'state/store';
+
+import {
+  IEvmTransaction,
+  IEvmTransactionsController,
+  ITransactionResponse,
+} from './types';
+import { getFormattedTransactionResponse } from './utils';
+
+const EvmTransactionsController = (): IEvmTransactionsController => {
+  const getInitialUserTransactions = async (): Promise<
+    ITransactionResponse[]
+  > => {
+    const { activeNetwork, accounts, activeAccount } = store.getState().vault;
+
+    const { address: userAddress } = accounts[activeAccount];
+
+    const {
+      label: networkLabel,
+      url: networkUrl,
+      chainId: networkChainId,
+      default: isNetworkDefault,
+      apiUrl: networkApiUrl,
+    } = activeNetwork;
+
+    const etherscanSupportedNetworks = [
```

Comment:
there's no need to have that here. Maybe something like a utils/constants file to get this array
```typescript
//utils.ts or constants.ts
const ETHER_SCRAN_SUPPORTED_NETWORKS = [
   'homestead',
   'ropsten',
   'rinkeby',
   'goerli',
   'kovan',
      ];
```